### PR TITLE
Replaces tesla engine with monkey engine

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
obraz

### Why is this change good for the game?
i am removing lag by removing lag inducing tesla

# Wiki Documentation
### Briefly describe your PR and the impacts of it, in layman's terms. 
Replaces stupid tesla engine room with a room full of ~treadmills~ stationary bikes and monkey cubes.

### What should players be aware of when it comes to the changes your PR is implementing?
That lag is gone

### What general grouping does this PR fall under? 
mmmunki

# Changelog
:cl:  
rscadd: adds a monkey powered engine
rscdel: removes tesla from engineering
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
I mean

**Comment by MegaEmpirical** [comment]
I can't tell if this is serious or not. Would be funny as an admin spawn.

**Comment by Cartlord** [comment]
For the unaware, this is a reference to another PR that I think is still ongoing - someone wanted to remove Tesla from Cargo on basically no grounds other than "badly-contained Teslas lag a lot", and someone jokingly suggested that every engine should be replaced with monkeys on bikes to get rid of the lag.

**Comment by ChesterTheCheesy** [comment]
@Cartlord who even are you

**Comment by Cartlord** [comment]
Why do you ask?

**Comment by Marmio64** [comment]
> Why do you ask?

because you're a random who recently showed up and started commenting and liking/disliking a bunch of PRs. if people know that someone is trustworthy their opinion will be valued more, so its good to know if someone is just a random from TG who recently started playing or a yoggite veteran

**Comment by patpol4** [comment]
Would probably be fine to merge if it was it's own engine disabled in config. Only for events etc.
But as a tesla replacement? come on man you didnt even change the filename

**Review by patpol4 — APPROVED** [review]
Make it a config disabled admin only engine or close the PR because this is just silly

**Review by patpol4 — CHANGES_REQUESTED** [review]
Meant to request changes

**Comment by ChesterTheCheesy** [comment]
High quality shitpost
